mongodb-atlas-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1205 @@
+        signals.cluster.diskReadCount.asTarget(),
+        signals.cluster.diskWriteCount.asTarget(),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription("The number of read and write I/O's processed.")
+      + g.panel.timeSeries.standardOptions.withUnit('iops')
+      + g.panel.timeSeries.options.legend.withPlacement('right')
+      + g.panel.timeSeries.options.legend.withAsTable(true),
+
+    hardwareIOWaitTime:
+      commonlib.panels.generic.timeSeries.base.new('Hardware I/O wait time / $__interval', targets=[
+        signals.cluster.diskReadTime.asTarget()
+        + g.query.prometheus.withInterval('2m'),
+        signals.cluster.diskWriteTime.asTarget()
+        + g.query.prometheus.withInterval('2m'),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription('The amount of time spent waiting for I/O requests.')
+      + g.panel.timeSeries.standardOptions.withUnit('ms')
+      + g.panel.timeSeries.options.tooltip.withSort('desc')
+      + g.panel.timeSeries.options.legend.withPlacement('right')
+      + g.panel.timeSeries.options.legend.withAsTable(true),
+
+    hardwareCPUInterruptServiceTime:
+      commonlib.panels.generic.timeSeries.base.new('Hardware CPU interrupt service time / $__interval', targets=[
+        signals.cluster.cpuIrqTime.asTarget()
+        + g.query.prometheus.withInterval('2m'),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription('The amount of time spent servicing CPU interrupts.')
+      + g.panel.timeSeries.standardOptions.withUnit('ms')
+      + g.panel.timeSeries.options.tooltip.withSort('desc'),
+
+    memoryUsed:
+      commonlib.panels.generic.timeSeries.base.new('Memory used', targets=[
+        signals.cluster.memoryResident.asTarget(),
+        signals.cluster.memoryVirtual.asTarget(),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription('The amount of RAM and virtual memory being used.')
+      + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+      + g.panel.timeSeries.options.tooltip.withSort('desc'),
+
+    diskSpaceUsage:
+      commonlib.panels.generic.timeSeries.base.new('Disk space usage', targets=[
+        signals.cluster.diskSpaceUtilization.asTarget(),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription('The percentage of hardware space used.')
+      + g.panel.timeSeries.standardOptions.withUnit('percentunit')
+      + g.panel.timeSeries.standardOptions.withMin(0)
+      + g.panel.timeSeries.standardOptions.withMax(1)
+      + g.panel.timeSeries.options.tooltip.withSort('desc'),
+
+    networkRequests:
+      commonlib.panels.generic.timeSeries.base.new('Network requests', targets=[
+        signals.cluster.networkRequests.asTarget(),
+      ])
+      + g.panel.timeSeries.panelOptions.withDescription('The number of distinct requests that the server has received.')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps')
+      + g.panel.timeSeries.options.tooltip.withSort('desc'),
+
+    networkThroughput:
```

Comment:
+1 for `last*`, and at least `mean` as additional data columns for a quick overview on the side